kuma/attachments/tests/conftest.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+
+import pytest
+from django.core.files.base import ContentFile
+
+from kuma.attachments.models import Attachment, AttachmentRevision
+
+
+@pytest.fixture
+def file_attachment(db, wiki_user):
+    file_id = 97
+    filename = 'test.txt'
+    title = 'Test text file'
+
+    attachment = Attachment(title=title, mindtouch_attachment_id=file_id)
+    attachment.save()
+    revision = AttachmentRevision(
+        title=title,
+        is_approved=True,
+        attachment=attachment,
+        mime_type='text/plain',
+        description='Initial upload',
+        created=datetime.datetime.now(),
+    )
+    revision.creator = wiki_user
+    revision.file.save(filename, ContentFile('This is only a test.'))
```

Comment:
This creates a new file "test.txt" for each test that uses this fixture. This isn't much worse than the old tests, but I suspect we could do better by making the actual file a session-scoped fixture, or even point it at an existing test file.  That may be a follow-on task.